ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_manipulation.py

```diff
@@ -1015,3 +1015,48 @@ def test_expand(
         x=x[0],
         shape=shape,
     )
+
+
+@st.composite
+def _concat_from_sequence_helper(draw):
+    dtypes, arrays, shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+            num_arrays=helpers.ints(min_value=1, max_value=6),
+            ret_shape=True,
+            min_num_dims=1,
+            shared_dtype=True,
+        )
+    )
+    axis = draw(helpers.get_axis(shape=shape))
+    return dtypes, arrays, axis
+
+
+# concat_from_sequence
+@handle_test(
+    fn_tree="functional.ivy.experimental.concat_from_sequence",
+    dtypes_arrays_axis=_concat_from_sequence_helper(),
+    new_axis=st.integers(min_value=0, max_value=1),
```

Comment:
I did this, now I am getting this error, I feel that the test case is sending a list of tuples as input arrays, and the decorator `@to_native_arrays_and_back` is not able to convert the tuples inside to native arrays.
```
E           TypeError: cat() received an invalid combination of arguments - got (list, dim=tuple), but expected one of:
E            * (tuple of Tensors tensors, int dim, *, Tensor out)
E            * (tuple of Tensors tensors, name dim, *, Tensor out)"
```

Reply:
Based on this error message I'd say the problem is `dim`. It says you're passing a tuple dim but `torch.cat` expects an int dim.

Reply:
I got the int dimension instead of tuple by doing `arrays = arrays[0]`. But the arrays which are getting generated by thre test function, aren't of the same shape, hence can't be concatenated. Is there a way to fix the shapes of arrays ?

Reply:
You shouldn't keep only the first element of arrays. Please undo that change. 
You need to set `force_int=True` in `get_axis` to make sure it doesn't return a tuple.